packages/theme-ui/test/layout.js

```diff
@@ -22,3 +25,9 @@ test.each([
   const json = renderer.create(<Component />).toJSON()
   expect(json).toMatchSnapshot()
 })
+
+test('forwards ref to div', () => {
+  const ref = React.createRef(null)
+  const tree = render(<Box ref={ref} />)
+  expect(ref.current.tagName).toBe('DIV')
```

Comment:
Cool, I was thinking about how to test the ref passing when doing a shallow rendering,,, just this easy... assert it directly :D